DigitalLearningSolutions.Web/Services/RegisterAdminService.cs

```diff
@@ -22,13 +23,23 @@ ICentresDataService centresDataService
             this.centresDataService = centresDataService;
         }
 
-        public bool IsRegisterAdminAllowed(int centreId)
+        public bool IsRegisterAdminAllowed(int centreId, int? loggedInUserId = null)
         {
-            var admins = userDataService.GetAdminsByCentreId(centreId);
+            var adminsAtCentre = userDataService.GetAdminsByCentreId(centreId).ToList();

```

Comment:
Can we rename `GetAdminsByCentreId`? It's not actually getting all the admins at the centre; it's getting all the active admin accounts at the centre which belong to active users.

Reply:
Is `GetActiveAdminsByCentreId` fine? I find it better than e.g. `GetActiveAdminsForActiveUsersByCentreId` which is more explicit but may also be a bit confusing

Reply:
Looks like HEE are going to deactivate all a user's centre accounts when they deactivate a user, so `GetActiveAdminsByCentreId` should be fine.